tests/test_csv__provider.py

```diff
@@ -95,15 +99,348 @@ def test_query(config):
 
 
 def test_get(config):
-    p = CSVProvider(config)
+    """Testing get method"""
 
+    p = CSVProvider(config)
     result = p.get('964')
     assert result['id'] == '964'
     assert result['properties']['value'] == '99.9'
 
 
 def test_get_not_existing_item_raise_exception(config):
     """Testing query for a not existing object"""
+
     p = CSVProvider(config)
     with pytest.raises(ProviderItemNotFoundError):
         p.get('404')
+
+
+# test on common comparisons operations
+def test_eq(config):
```

Comment:
It is better to change the name to something that gives more context to cql testing, like `test_cql_eq` in this case

The comment is still valid for all the functions’ name below